packer/install-dependencies.sh

```diff
@@ -62,8 +62,8 @@ installGPUDrivers
 echo "  - nvidia-docker2 nvidia-container-runtime" >> ${VHD_LOGS_FILEPATH}
 
 VNET_CNI_VERSIONS="
+1.0.30
 1.0.29
-1.0.28
```

Comment:
Let's keep any changes to the `VNET_CNI_VERSIONS` to the Azure CNI PR

Reply:
oops, fixed it